other/materials_designer/create_cluster_sphere.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    DEFAULT_SLAB_PARAMETERS = {
</pre><p>Let's explain above that we align the Z axis of the cluster with a direction defined by the miller indices.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>